templates/components/form/header.html.twig

```diff
@@ -55,108 +52,21 @@
    {% endif %}
 {% endif %}
 
-{% if item.canEdit(item.fields['id']) %}
-<form name="massaction_{{ rand }}" id="massaction_{{ rand }}" method="post"
-      action="{{ path('/front/massiveaction.php') }}" data-submit-once>
-   <div id="massive_container_{{ rand }}"></div>
-   <input type="hidden" name="_glpi_csrf_token" value="{{ csrf_token() }}" />
-</form>
+{% set no_header = no_header|default(false) %}
+{% set open_form = no_header or item.isNewID(item.fields['id']) or in_twig is not defined %}
 
+{% if open_form and item.canEdit(item.fields['id']) %}
 <form name="asset_form" method="post" action="{{ target }}" {{ formoptions|raw }} enctype="multipart/form-data" data-submit-once>
    <input type="hidden" name="entities_id" value="{{ entity_id }}" />
    {% if _request['_in_modal'] is defined and _request['_in_modal'] == "1" %}
       <input type="hidden" name="_in_modal" value="1"/>
    {% endif %}
 {% endif %}
    <div id="mainformtable">
-      {% set template_name = item.fields['template_name']|verbatim_value %}
-      {% if withtemplate == 2 and not item.isNewItem() %}
-         <input type="hidden" name="template_name" value="{{ template_name }}" />
-         {% set nametype = __('Created from the template %s')|format(template_name) %}
-      {% elseif withtemplate == 1 %}
-         <input type="hidden" name="is_template" value="1" />
-      {% elseif item.isNewItem() %}
-         {% set nametype = __('%1$s - %2$s')|format(__('New item'), nametype) %}
+      {% if no_header == false and (in_twig is defined or _get._in_modal|default(false)) %}
+         {% include 'components/form/header_content.html.twig' with {inside_main: true} %}
```

Comment:
Please use the `include` function, as recommanded in [Twig docs](https://twig.symfony.com/doc/3.x/tags/include.html).
```suggestion
         {{ include('components/form/header_content.html.twig', {'inside_main': true}) }}
```